packages/enclave-portal/src/components/Header.tsx

```diff
@@ -141,7 +145,7 @@ const headerLinks = [
   },
 ];
 
-const externalAppLinks = [
+const externalAppLinks: HeaderLinkItem[] = [
```

Comment:
fyi, the enclave portal not have the apps menu.  there'll be a ticket to remove it later.